docs/dev/table/hive/index.md

```diff
@@ -281,6 +261,11 @@ Please find the required dependencies for different Hive major versions below.
 </div>
 </div>
 
+If you use the hive version of HDP or CDH, you need to refer to the dependency in the previous section and select a similar version.
+
+And you need to specify selected and supported "hive-version" in yaml, HiveCatalog and HiveModule.
```

Comment:
I think providing an example for this would be more helpful

Reply:
I don't want provide too detailed information. This is the home page of hive. These words should be enough.